internal/compiler/typeregister.rs

```diff
@@ -161,6 +161,16 @@ pub fn reserved_property(name: &str) -> PropertyLookupResult {
             }
         }
     }
+
+    if name == "init" {
```

Comment:
Instead of putting it there, add it to the `reserved_properties()` iterator.
Then it should also auto-complete correctly in the LSP.